Change all occurences of @strong{Note:} to avoid makeinfo warnings and
spurious cross references. This is a backport of the following change:

2003-09-26  Richard M. Stallman  <rms@gnu.org>

	* buffers.texi, commands.texi, debugging.texi, eval.texi:
	* loading.texi, minibuf.texi, text.texi, variables.texi:
	Avoid @strong{Note:}.

--- elisp-orig/buffers.texi	2001-09-23 18:52:16.000000000 +0000
+++ elisp/buffers.texi	2008-11-25 22:20:16.000000000 +0000
@@ -107,5 +107,5 @@
 that, you must use the functions described in @ref{Displaying Buffers}.
 
-  @strong{Note:} Lisp functions that change to a different current buffer
+  @strong{Warning:} Lisp functions that change to a different current buffer
 should not depend on the command loop to set it back afterwards.
 Editing commands written in Emacs Lisp can be called from other programs
--- elisp-orig/commands.texi	2001-09-10 05:06:03.000000000 +0000
+++ elisp/commands.texi	2008-11-25 22:20:16.000000000 +0000
@@ -2018,5 +2018,5 @@
 function.
 
-@strong{Note:} Don't bind this variable with @code{let}.  It is often
+@strong{Warning:} Don't bind this variable with @code{let}.  It is often
 buffer-local, and if you bind it around reading input (which is exactly
 when you @emph{would} bind it), switching buffers asynchronously while
--- elisp-orig/debugging.texi	2000-12-04 16:15:18.000000000 +0000
+++ elisp/debugging.texi	2008-11-25 22:20:16.000000000 +0000
@@ -216,5 +216,5 @@
 @code{debug-on-entry} always returns @var{function-name}.
 
-@strong{Note:} if you redefine a function after using
+@strong{Warning:} if you redefine a function after using
 @code{debug-on-entry} on it, the code to enter the debugger is discarded
 by the redefinition.  In effect, redefining the function cancels
--- elisp-orig/eval.texi	2000-01-05 15:17:53.000000000 +0000
+++ elisp/eval.texi	2008-11-25 22:20:16.000000000 +0000
@@ -589,5 +589,5 @@
 values.  Loading a file also does evaluation (@pxref{Loading}).
 
-  @strong{Note:} it is generally cleaner and more flexible to store a
+  It is generally cleaner and more flexible to store a
 function in a data structure, and call it with @code{funcall} or
 @code{apply}, than to store an expression in the data structure and
--- elisp-orig/loading.texi	2001-09-06 19:49:45.000000000 +0000
+++ elisp/loading.texi	2008-11-25 22:20:16.000000000 +0000
@@ -149,5 +149,5 @@
 functions should use @code{read}.
 
-@strong{Note:} Instead of using this variable, it is cleaner to use
+Instead of using this variable, it is cleaner to use
 another, newer feature: to pass the function as the @var{read-function}
 argument to @code{eval-region}.  @xref{Eval}.
--- elisp-orig/minibuf.texi	2001-10-02 02:34:48.000000000 +0000
+++ elisp/minibuf.texi	2008-11-25 22:20:16.000000000 +0000
@@ -1064,6 +1064,6 @@
 case, point goes at the beginning of @var{initial}.  The default for
 @var{initial} is @code{nil}---don't insert any file name.  To see what
-@var{initial} does, try the command @kbd{C-x C-v}.  @strong{Note:} we
-recommend using @var{default} rather than @var{initial} in most cases.
+@var{initial} does, try the command @kbd{C-x C-v}.  @strong{Please note:}
+we recommend using @var{default} rather than @var{initial} in most cases.
 
 Here is an example: 
--- elisp-orig/text.texi	2001-10-02 02:21:59.000000000 +0000
+++ elisp/text.texi	2008-11-25 22:20:16.000000000 +0000
@@ -3694,5 +3694,5 @@
 the body of a @code{combine-after-change-calls} form.
 
-@strong{Note:} If the changes you combine occur in widely scattered
+@strong{Warning:} If the changes you combine occur in widely scattered
 parts of the buffer, this will still work, but it is not advisable,
 because it may lead to inefficient behavior for some change hook
--- elisp-orig/variables.texi	2001-09-06 19:43:44.000000000 +0000
+++ elisp/variables.texi	2008-11-25 22:20:16.000000000 +0000
@@ -1290,5 +1290,5 @@
 Displays}.
 
-@strong{Note:} Do not use @code{make-local-variable} for a hook
+@strong{Warning:} Do not use @code{make-local-variable} for a hook
 variable.  Instead, use @code{make-local-hook}.  @xref{Hooks}.
 @end deffn
